contracts/libs/Owned.sol

```diff
@@ -4,17 +4,15 @@ pragma solidity 0.4.11;
 contract Owned {
     address public owner;
     modifier onlyOwner() {
-        if (!isOwner(msg.sender)) {
-            throw;
-        }
+        require(isOwner(msg.sender));
         _;
     }
 
     function Owned() { owner = msg.sender; }
 
     function isOwner(address addr) public returns(bool) { return addr == owner; }
 
-    function transfer(address _owner) onlyOwner {
+    function transfer(address _owner) public onlyOwner {
```

Comment:
What does it mean for a function to be `public`?

Reply:
Public is the default (for now atleast). Best practices states that we should explicitly state the visibility.
http://solidity.readthedocs.io/en/develop/miscellaneous.html?highlight=visibility#function-visibility-specifiers